packages/flutter/lib/src/services/text_formatter.dart

```diff
@@ -88,6 +88,9 @@ enum MaxLengthEnforcement {
 ///  * [FilteringTextInputFormatter], a provided formatter for filtering
 ///    characters.
 abstract class TextInputFormatter {
+  /// Allows subclasses to have const constructor
```

Comment:
nit: `This constructor enables subclasses to provide const constructors so that they can be used in const expressions.`